packages/assets-controllers/src/NftController.ts

```diff
@@ -520,6 +520,12 @@ export class NftController extends BaseController<
     });
   }
 
+  #getNftCollectionApi(): string {
+    // False negative.
+    // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
```

Comment:
Do you know if this line is required?

Reply:
Unfortunately yes, a thread here https://github.com/MetaMask/core/pull/4506#discussion_r1672210804